Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java

```diff
@@ -78,38 +86,33 @@ protected void testGetEventIdentificationType(LogEventRequestType request, Strin
         Boolean isRequesting) {
 
         EventIdentificationType eventIdentificationType = request.getAuditMessage().getEventIdentification();
-        if (isRequesting) {
-            assertEquals(getAuditTransforms().getServiceEventActionCodeRequestor(),
-                eventIdentificationType.getEventActionCode());
-        } else {
-            assertEquals(getAuditTransforms().getServiceEventActionCodeResponder(),
-                eventIdentificationType.getEventActionCode());
-        }
-        assertEquals(AuditTransformsConstants.EVENT_OUTCOME_INDICATOR_SUCCESS.toString(), eventIdentificationType
-            .getEventOutcomeIndicator().toString());
-        if (isRequesting) {
-            assertEquals(getAuditTransforms().getServiceEventIdCodeRequestor(),
-                eventIdentificationType.getEventID().getCode());
-        } else {
-            assertEquals(getAuditTransforms().getServiceEventIdCodeResponder(),
-                eventIdentificationType.getEventID().getCode());
-        }
-        assertEquals(getAuditTransforms().getServiceEventCodeSystem(),
-            eventIdentificationType.getEventID().getCodeSystemName());
+        String eventActionCode;
+        String eventIdCode;
+        String eventIdDisplayName;
 
         if (isRequesting) {
-            assertEquals(getAuditTransforms().getServiceEventDisplayRequestor(), eventIdentificationType.getEventID()
-                .getDisplayName());
+            eventActionCode = getAuditTransforms().getServiceEventActionCodeRequestor();
+            eventIdCode = getAuditTransforms().getServiceEventIdCodeRequestor();
+            eventIdDisplayName = getAuditTransforms().getServiceEventDisplayRequestor();
         } else {
-            assertEquals(getAuditTransforms().getServiceEventDisplayResponder(), eventIdentificationType.getEventID()
-                .getDisplayName());
+            eventActionCode = getAuditTransforms().getServiceEventActionCodeRequestor();
```

Comment:
Isn't this incorrect? below details for eventActionCode,eventIdCode and eventIdDisplayName should be of  responder details not requestor details. For example using getServiceEventDisplayResponder()